app/views/groups/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="containers" style= "padding: 0.5rem; max-width: 700px; margin: 0rem auto;">
+  <h5 class='text-center display-6'>TRANSACTION</h5>
+  <div class="container p-4 pd-0 d-flex justify-content-between">
+		<h5 class="fw-bold">Total Amount:</h5>
+		<p class="fw-bold">$<%= @group.total_amount.round(2) %></p>
+  </div>
+  <div class="p-4 pt-0">
+    <%= image_tag @group.image, class: 'img-thumbnail', style: 'max-width: 200px; max-height: 200px;' %>
+    <p class="text-muted text-end pt-2"><%= @group.name %></p>
+  </div>
+</div>
+  <div class="divider mb-2"></div>
+	  <%= render 'transactions/index', locals: { transactions: @transactions } %>
+  </div>
+
+<footer class="footer" style= "padding: 0.5rem; max-width: 700px; margin: 0rem auto;">
+  <div class="d-grid gap-2">
+    <%= link_to "Add new transaction", new_transaction_path, class: "btn btn-success border-radius-none" %>
+  </div>
```

Comment:
You did an amazing job 🥇 @ngeelani48, however, you need to fix some points 😿.
- [ ]  As I can see when I'm creating a new category, it's taken to the "transaction" page on success, Kindly, make sure when the user clicks (or taps) the "Save" button to create the new category, and it's taken to the home page on success
- [ ] I could be better if you follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as it is required, including:
* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements.

It could be better if you give your button a `position fix` to the bottom, to stitch the design as much as possible 👍 

> | | Original design | | Your project | |
> |-|---------|-|--------|-|
> | |![transactions_list](https://github.com/ngeelani48/budget_manager/assets/57408419/84f51842-d996-4eaf-aa92-a5be9f355ccd)| |![Screenshot 2023-07-06 183319](https://github.com/ngeelani48/budget_manager/assets/57408419/b6a5eac7-4812-4be7-8610-1abd4a18906c)| |

Reply:
Hello @joel-kalema, 
Thanks for the review and your time. 🌷 
I will do my best to fix the issues that you have pointed out.

Thanks again and happy coding ... 👍 👍 👍

Reply:
Thank you @ngeelani48, You Did good job 💪 
Cheers, and Happy coding!👏👏👏